### ISSUE TITLE
Negative filter for editor

### ISSUE THREAD

**Issue by naoliv** [mapbox/osmcha-frontend#246]
## *Brief Description*
There should be a way to filter the editor in using a negative (or "inverse") expression.

## What is the motivation / use case for this feature?
For example, if we want to filter all the editions made with JOSM we can simply use `josm`
But we can't filter all the editions that were not made with JOSM (eg, something like a `-josm`)

**Comment by robybully** [comment]
I'm also waiting for this feature. I want to ignore some users or editors in my filtered area of interest.

**Comment by andrewharvey** [comment]
This would be really useful to filter out StreetComplete edits, which can create a ton of changesets, and the damage one can do through StreetComplete is more limited so one might want to just skip reviewing all these edits.

**Comment by zorglubu** [comment]
> This would be really useful to filter out StreetComplete edits, which can create a ton of changesets, and the damage one can do through StreetComplete is more limited so one might want to just skip reviewing all these edits.

I totally agree. I was going to ask the same thing : the ability to filter out StreetComplete edits, in order to focus on real risks.

**Comment by Lee-Carre** [comment]
To avoid proliferation of almost-identical issues being opened for what is essentially the same feature, I'll add to this discussion.

Besides being able to exclude (filter out) an arbitrary set of editors, like the OP requests, I would like the ability to exclude sets based on hashtag (e.g. #France).

See the discussion of [OSM changeset #114157461](http://www.openstreetmap.org/changeset/114157461).

**Comment by Lee-Carre** [comment]
@robybully

> I want to ignore some users …

You could add said users to your account's list of trusted users, and then configure the filtering to hide changesets by trusted users.

I use this technique to hide my own changesets when reviewing my local area (since I know what I changed, I want to see what was changed by others). Should be quite adaptable to hide an arbitrary set of users, until more expressive matching is implemented.

**Comment by mfrasca** [comment]
I'm reviewing edits related to an organised edit that did not use hashtags, and in the area there's page after page of just StreetComplete changesets, which I assume to be fine and unrelated to what I am trying to find.  I looked for how to specify a negative filter for the editor, and find this open issue.

**Comment by Filip10** [comment]
I'm also looking for this feature because I'm checking new users. StreetComplete changes are many and not worth to check, because of low probability to be bad.
